python/cuml/test/test_random_forest.py

```diff
@@ -143,7 +150,64 @@ def test_rf_regression(datatype, use_handle, split_algo,
                          random_state=10)
         sk_model.fit(X_train, y_train)
         sk_predict = sk_model.predict(X_test)
-        sk_mse = mean_squared_error(y_test, sk_predict)
+        sk_r2 = r2_score(y_test, sk_predict)
 
         # compare the accuracy of the two models
-        assert cu_mse <= (sk_mse + 0.07)
+        assert cu_r2 >= (sk_r2 + 0.07)
+
+
+@pytest.mark.parametrize('datatype', [np.float32])
+def test_rf_classification_default(datatype):
+
+    train_rows = np.int32(30*0.8)
+    X, y = make_classification(n_samples=30, n_features=10,
```

Comment:
Does this need to differ from other `tiny_classification` tests that use 20 samples?